### ISSUE TITLE
Cannot remove user from organization from members page

### ISSUE THREAD

**Issue by classabbyamp** [go-gitea/gitea#11628]
- Gitea version (or commit ref): 1.11.5
- Git version: 2.24.3
- Operating system: Docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:
```
2020-05-26T03:00:18.664585502Z [Macaron] 2020-05-26 03:00:18: Started POST /org/{{ org name }}/members/action/remove for {{ ip }}
```

## Description

When I try to remove an organization member from the org members page (`/org/{{ org name }}/members`), the request never completes, and other requests start to freeze for other users. This does not happen when removing the member from the `/org/{{ org name }}/teams/{{ team name }}` page.

**Comment by zeripath** [comment]
This is a duplicate of the already fixed issue #10549 

#11438 and its backport #11439 fix this issue for master/1.12 and 1.11 respectively.

**Comment by argentini** [comment]
Also, it happens when I sign in as the user and try to leave the organization.
